Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\r\n# Created by https://www.gitignore.io/api/macos\r\n\r\n### macOS ###\r\n*.DS_Store\r\n.AppleDouble\r\n.LSOverride\r\n\r\n# Icon must end with two \\r\r\nIcon\r\n\r\n\r\n# Thumbnails\r\n._*\r\n\r\n# Files that might appear in the root of a volume\r\n.DocumentRevisions-V100\r\n.fseventsd\r\n.Spotlight-V100\r\n.TemporaryItems\r\n.Trashes\r\n.VolumeIcon.icns\r\n.com.apple.timemachine.donotpresent\r\n\r\n# Directories potentially created on remote AFP share\r\n.AppleDB\r\n.AppleDesktop\r\nNetwork Trash Folder\r\nTemporary Items\r\n.apdisk\r\n\r\n# End of https://www.gitignore.io/api/macos\r\n\r\n# debug file\r\n_site
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 71dfda2ea65fa25a9951e6f7329c8bd37bd9ffbf)
+++ b/.gitignore	(date 1774662518212)
@@ -6,6 +6,8 @@
 .AppleDouble
 .LSOverride
 
+./.idea
+
 # Icon must end with two \r
 Icon
 
